sandbox/lsm/lsm.go

```diff
@@ -0,0 +1,168 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2025 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package lsm
+
+import (
+	"encoding/binary"
+	"syscall"
+	"unsafe"
+
+	"golang.org/x/sys/unix"
+
+	"github.com/snapcore/snapd/arch"
+)
+
+const (
+	// https://elixir.bootlin.com/linux/v6.14/source/include/uapi/linux/lsm.h#L44
+	LSM_ID_UNDEF      = 0
+	LSM_ID_CAPABILITY = 100
+	LSM_ID_SELINUX    = 101
+	LSM_ID_SMACK      = 102
+	LSM_ID_TOMOYO     = 103
+	LSM_ID_APPARMOR   = 104
+	LSM_ID_YAMA       = 105
+	LSM_ID_LOADPIN    = 106
+	LSM_ID_SAFESETID  = 107
+	LSM_ID_LOCKDOWN   = 108
+	LSM_ID_BPF        = 109
+	LSM_ID_LANDLOCK   = 110
+	LSM_ID_IMA        = 111
+	LSM_ID_EVM        = 112
+	LSM_ID_IPE        = 113
+
+	lsmCount = LSM_ID_IPE - LSM_ID_CAPABILITY + 1
+
```

Comment:
Could you end the previous const block here and open a new one. `go doc` formats this more nicely. Put the comments just above the const line.